### DISCUSSION TITLE
Cannot find module 'private-next-rsc-mod-ref-proxy' error for Vercel Analytics

### DISCUSSION THREAD

**Discussion by benthecoder** [vercel/community#2117] [Help] [answered]
### Summary

I was following this, and it doesn't work. I'm getting this error.

<img width="964" alt="image" src="https://user-images.githubusercontent.com/49143413/232869661-4bcd8df6-166a-407c-a6b3-40663702a0d5.png">

**Comment by amyegan** [comment]
Hi @benthecoder! I think you forgot to include a link in your summary. Can you share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) or link to the guide or tutorial you followed?

**Reply by benthecoder** [reply, marked as answer]
Thanks @amyegan  for the response but I solved the issue already by adding this wrapper https://github.com/benthecoder/blog/blob/main/components/analytics.tsx

But I'm still curious why the tutorial never gave this suggestion.

**Reply by ritajeong** [reply]
@benthecoder 
Hi! I have the same problem. May I ask which wrapper it is? I'm getting a 404 on that link.

**Reply by ritajeong** [reply]
Oh, I created a component with analytics and added it to the layout file (I'm using the app directory)
```
"use client";
import { Analytics } from "@vercel/analytics/react";

const VercelAnalytics = () => {
  return <Analytics />;
};

export default VercelAnalytics;
```
